exporter/tanzuobservabilityexporter/README.md

```diff
@@ -40,42 +44,56 @@ exporters:
 service:
   pipelines:
     traces:
-      receivers: [examplereceiver]
-      processors: [batch]
-      exporters: [tanzuobservability]
+      receivers: [ examplereceiver ]
+      processors: [ batch ]
+      exporters: [ tanzuobservability ]
     metrics:
-      receivers: [examplereceiver]
-      processors: [batch]
-      exporters: [tanzuobservability]
+      receivers: [ examplereceiver ]
+      processors: [ batch ]
+      exporters: [ tanzuobservability ]
 ```
 
 ## Advanced Configuration
 
 ### Resource Attributes on Metrics
 
-Client programs using an OpenTelemetry SDK can be configured to wrap all emitted telemetry (metrics, spans, logs) with 
-a set of global key-value pairs, called [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md).
+Client programs using an OpenTelemetry SDK can be configured to wrap all emitted telemetry (metrics, spans, logs) with
+a set of global key-value pairs,
+called [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
+.
 By default, the Tanzu Observability Exporter includes resource attributes on spans but _excludes_ them on metrics. To
-include resource attributes as tags on metrics, configure the `resource_attributes` stanza per the example below.
+include resource attributes as tags on metrics, set the flag `resource_attrs_included` to `true` as per the example
+below.
 
 **Note:** Tanzu Observability has a 254-character limit on tag key-value pairs. If a resource attribute exceeds this
 limit, the metric will not show up in Tanzu Observability.
 
+### Application Resource Attributes on Metrics
+
+The Tanzu Observability Exporter will include application resource attributes on metrics(`application`, `service.name`
```

Comment:
1. Let's make the words "application resource attributes" link to https://docs.wavefront.com/trace_data_details.html#application-tags, since may not know what makes these four attributes special.
2. A space is missing in "metrics(".
3. Add a sentence explaining that `service.name` becomes `service` on the transformed metric.

Reply:
making the change